github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1147,7 +1147,7 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.24-ipv6-sig-network
+          value: k8s-1.25-ipv6-sig-network
```

Comment:
@brianmcarey 
Is there a way to create env var (either in the yaml of in the bash scripts) that depend on the job name ?
this way we will need to change only the `name` field

Reply:
Thanks, do you like this idea ?
if so we can try on follow PR